test-requirements.txt

```
@@ -1,7 +1,10 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+lightkube==0.11.0
 black
 flake8 < 4.1
 jinja2 < 3.1
```

I thought this would trigger our format checking, but these should be alphabetical.  
```suggestion
black
flake8 < 4.1
jinja2 < 3.1
lightkube==0.11.0
```

---

Done.